sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -2911,6 +2912,21 @@ class DataSourceV2SQLSuite
     }
   }
 
+  test("ShowCatalogs") {
```

Comment:
I would add the JIRA id as a prefix:

```suggestion
  test("SPARK-35973: ShowCatalogs") {
```